app.json

```diff
@@ -24,7 +24,11 @@
     "ios": {
       "supportsTablet": true,
       "buildNumber": "0.20.02.4",
-      "bundleIdentifier": "io.rain.app"
+      "bundleIdentifier": "io.rain.app",
+      "infoPlist": {
```

Comment:
Is this needed?

Reply:
this one and datePickerCon: { backgroundColor: 'black', } need to fix showing calendar on ios. But I only one can enough. But I added two things because I have no devices to test this. u can remove if one option if work.

Reply:
I have tested it by removing the infoPlist and the date picker component still works. Please go ahead with the removal.